user_docs/en/changes.md

```diff
@@ -12,6 +12,7 @@
 
 ### Bug Fixes
 
+* Diff match patch proxy crash no longer causes NVDA to freeze. (#15850, #16027, @Danstiv)

```

Comment:
Markdown is generated during the build process, but shouldn't be checked in. Please modify `changes.t2t` (in the same directory as this file) and uncommit this file.

```suggestion
- Improved reliability of automatic text readout, particularly in terminal applications. (#15850, #16027, @Danstiv)

```

Reply:
@codeofdusk Please review #16492. There are no more `t2t` files in NVDA. The usage of `md` files is correct.